.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/de/57e61bb7cca6afdb1278dec468039ee17ad93d1f3803563e1ec39b2f7bf540

```diff
@@ -0,0 +1,39 @@
+I"�<h1 id="contributing-to-the-ict-baseline-alignment-framework">Contributing to the ICT Baseline Alignment Framework</h1>
+<h2 id="how-you-can-help">How you can help</h2>
+<h3 id="developing-and-evaluating-test-cases">Developing and Evaluating Test Cases</h3>
+<p>Need content from people doing this work. What do you need?</p>
+
+<p>I have a note about issuing templates. Does that belong bundled with this topic?</p>
```

Comment:
this section isn't ready for publishing

Reply:
Unfortunately, it is already published. There was no actual change to this file.

Reply:
how about change it to 'TBD' until we can put up something better?